src/controllers/user/createUser.ts

```diff
@@ -20,7 +20,13 @@ const dataSchema = z.object({
         invalid_type_error: "phNo should be a number",
         required_error: "phNo is a required parameter"
       })
-      .min(0, {
+      .int({
+        message: "phNo must be an integer"
+      })
+      .gte(0, {
+        message: "phNo must be valid"
+      })
+      .lte(99999999999999, {
```

Comment:
why is this so big XD

Reply:
Phone number was originally created as a column of width 14. I just used the same thing for validation.

Reply:
why was it 14